build/media-suite_compile.sh

```diff
@@ -608,7 +608,7 @@ if [[ $standalone = y ]] && enabled libopus; then
     if do_vcs "https://github.com/xiph/libopusenc.git"; then
         do_uninstall "${_check[@]}"
         do_autogen
-        do_separate_confmakeinstall --disable-{examples,doc}
+        do_separate_confmakeinstall --disable-{http,examples,doc}
```

Comment:
libopusenc doesn't have http.

Reply:
Then treat this PR as a mistake and do what's best. I'm reaching my limits in understanding how to use github.com features...

Reply:
Yeah I realized that it didn't have http as well after I took some time to continue my thoughts on the matter. Still learning.